owasp-dependency-check-suppressions.xml

```diff
@@ -156,6 +156,28 @@
       <cve>CVE-2018-1320</cve>
       <cve>CVE-2019-0205</cve>
   </suppress>
+  <suppress>
+    <!-- TODO: Fix by using com.datastax.oss:java-driver-core instead of com.netflix.astyanax:astyanax in extensions-contrib/cassandra-stroage -->
```

Comment:
Can you create a github issue for this so we don't forget.

Reply:
Created https://github.com/apache/druid/issues/10358